server/api/src/rsDataObjUnlink.cpp

```diff
@@ -356,11 +381,9 @@ int rsMvDataObjToTrash(
                 addKeyVal(&dataObjUnlinkInp->condInput, RESC_HIER_STR_KW, hier.c_str());
             }
             catch (const irods::exception& e) {
-                std::stringstream msg;
-                msg << "failed in irods::resolve_resource_hierarchy for [";
-                msg << dataObjUnlinkInp->objPath << "]";
                 irods::log(PASS(irods::error(e)));
-                // TODO: If the force flag is set, we just continue?
+                // TODO: If the force flag is set, we just continue???
+                // TODO: See irods/irods#3154 and talk to JMC
```

Comment:
maybe... just the issue number is sufficient :)

Reply:
lol Forgot I wrote this. Done